TestUtils/src/main/java/com/braintreepayments/api/testutils/MockBraintreeClientBuilder.java

```diff
@@ -128,6 +135,7 @@ public BraintreeClient build() {
         when(braintreeClient.getManifestActivityInfo(any())).thenReturn(activityInfo);
         when(braintreeClient.launchesBrowserSwitchAsNewTask()).thenReturn(
                 launchesBrowserSwitchAsNewTask);
+        when(braintreeClient.isPayPalInstalled()).thenReturn(isPayPalInstalled);
```

Comment:
Feel free to ignore [this comment](https://github.com/braintree/braintree_android/pull/1092/files#r1704227805) if it makes mocking more difficult. I'd rather get this merged in now, and we'll look into making the code more Kotlin-like in the future.